### PR TITLE
fix: use throttling-wired octokit everywhere

### PR DESCRIPTION
**Pull request by varl**
Related to #192.

This adds the throttling plugin to other usages of octokit that were overlooked the first time around.

cc: @Andarist

### CODE REVIEW

**Comment by varl** [comment]
Ugh, test broke because the mock for `getOctokit` isn't used. Looking into a fix.

File: src/run.ts [hunk 1]
```diff
@@ -26,6 +26,40 @@ import type {} from "@octokit/plugin-throttling/dist-types/types.d";
 // To avoid that, we ensure to cap the message to 60k chars.
 const MAX_CHARACTERS_PER_MESSAGE = 60000;
 
+const setupOctokit = (githubToken) => {
```
Comment (Andarist) [inline src/run.ts:29]: oh, now it makes sense... I totally missed the fact that each command~ has its own octokit instance

**Comment by Andarist** [comment]
@varl let me know if you need help with that

File: src/run.ts (hunk 1 above)
Comment (varl) [reply src/run.ts:29]: Same. Want me to refactor so we have a single instance for the entire module ?

**Comment by varl** [comment]
@Andarist cheers, test fixed in [e70152d](https://github.com/changesets/action/pull/291/commits/e70152dfbddc3b4d3973450a31fe9b862ae35d95). If you know of a better way, feel free to change it.

Since `github.getOctokit` isn't used anymore, I refactored the type to `GitHub` from '@actions/github/lib/utils' as used here: https://github.com/actions/toolkit/blob/main/packages/github/src/github.ts#L19

Hope that works the same way as before since it's the same instance type.

**Pull request merged**